Benchmarks build issue

>@ethereumjs/vm@5.9.1 build
> ../../config/cli/ts-build.sh
[Node build] Using tsconfig.prod.json
> tsc --build ./tsconfig.prod.json
[Node build] Working... DONE
Error: tsconfig.benchmarks.json(6,5): error TS5069: Option 'declarationMap' cannot be specified without specifying option 'declaration' or option 'composite'.
npm ERR! Lifecycle script `build:benchmarks` failed with error: 
npm ERR! Error: command failed 
npm ERR!   in workspace: @ethereumjs/vm@5.9.1 
npm ERR!   at location: /home/runner/work/ethereumjs-monorepo/ethereumjs-monorepo/packages/vm 
Error: Process completed with exit code 1

We're now seeing this build issue for the `vm` benchmarks CI script, most likely introduced with #1992. This should be resolved before the beta release.

--- a/packages/vm/tsconfig.benchmarks.json
+++ b/packages/vm/tsconfig.benchmarks.json
@@ -3,7 +3,7 @@
   "compilerOptions": {
     "lib": ["dom"],
     "sourceMap": false,
-    "declaration": false,
+    "declaration": true,
     "esModuleInterop": true
   },
   "include": ["benchmarks/**.ts"]

